outputs/alertmanager.go

```diff
@@ -83,6 +94,20 @@ func newAlertmanagerPayload(falcopayload types.FalcoPayload, config *types.Confi
 	if config.Alertmanager.ExpiresAfter != 0 {
 		amPayload.EndsAt = falcopayload.Time.Add(time.Duration(config.Alertmanager.ExpiresAfter) * time.Second)
 	}
+	for label, value := range config.Alertmanager.ExtraLabels {
+		if !regProm.MatchString(label) {
```

Comment:
the extra labels/annotations are static, we should check if they match the regexp in config.go, when we map the values, no need to check at each event.